integration_tests/mockApis/referrals.ts

```diff
@@ -29,13 +29,23 @@ export default {
     organisationId: string
     referralSummaries: Array<ReferralSummary>
     queryParameters?: Record<string, { equalTo: string }>
-  }): SuperAgentRequest =>
-    stubFor({
+    totalElements?: number
+    totalPages?: Paginated<ReferralSummary>['totalPages']
+  }): SuperAgentRequest => {
+    const pageNumber = Number(args.queryParameters?.page?.equalTo || 0)
+    const pageSize = 15
+    const totalPages = args.totalPages || 1
+    const totalElements = Object.prototype.hasOwnProperty.call(args, 'totalElements')
```

Comment:
Is this accounting for `totalElements` being `null`? So would then be `0`?

Reply:
Would there be a case when we pass in `null` as the optional `totalElements` argument to this stub method?

Reply:
Sorry don't think I phrased my question correctly, I was just curious about the use of `Object.prototype.hasOwnProperty.call(args, 'totalElements')` instead of `args.totalElements`.

Reply:
Ah, it might be what you were suggesting: basically, I want to accept `0` but not `undefined`

Reply:
Not really a use case for `0` yet, but just in case!